gateway-ha/src/test/java/io/trino/gateway/ha/HaGatewayTestUtils.java

```diff
@@ -104,6 +102,15 @@ public static File buildGatewayConfig(PostgreSQLContainer postgreSqlContainer, i
         return target;
     }
 
+    public static Map<String, String> buildPostgresVars(PostgreSQLContainer postgresql)
+    {
+        return ImmutableMap.<String, String>builder()
+                .put("POSTGRESQL_JDBC_URL", postgresql.getJdbcUrl())
+                .put("POSTGRESQL_USER", postgresql.getUsername())
+                .put("POSTGRESQL_PASSWORD", postgresql.getPassword())
+                .build();
```

Comment:
We should use `buildOrThrow` method when building `ImmutableMap`. `build` method will be banned in the future.